src/extensions/default/CSSCodeHints/main.js

```diff
@@ -468,7 +472,7 @@ define(function (require, exports, module) {
 
     AppInit.appReady(function () {
         var cssPropHints = new CssPropHints();
-        CodeHintManager.registerHintProvider(cssPropHints, ["css", "scss", "less"], 0);
+        CodeHintManager.registerHintProvider(cssPropHints, ["css", "scss", "less"], 1);
```

Comment:
@swmitra Sorry I already forgot about the opening note of PR 😸 thanks for the detailed explanation!